lib/services/livesync/livesync-service.ts

```diff
@@ -671,6 +672,10 @@ export class LiveSyncService extends EventEmitter implements IDebugLiveSyncServi
 		}
 	}
 
+	public emitLivesyncEvent (event: string, livesyncData: ILivesyncEventData): boolean {
```

Comment:
is this only a rename of the function?

Reply:
Not exactly. There is still a calls to `this.emit` that made the introduction of ILivesyncEventData very ugly(all of the properties had to be optional).

Reply:
and why was the introduction of ILivesyncEventData necessary?

Reply:
So that we have types of the properties of the data object. I was going to miss it and break Sidekick.